commit bf2749ab998446f13ba20c8cbc33a8c4d1c36963
Author: Jean-Baptiste Mardelle <jb@kdenlive.org>
Date:   Sun Feb 5 22:54:50 2017 +0100

    Fix crash in ClipInfo::update method when clip has no resource

diff --git a/src/mlt++/MltPlaylist.cpp b/src/mlt++/MltPlaylist.cpp
index 5676060..111d3f5 100644
--- a/src/mlt++/MltPlaylist.cpp
+++ b/src/mlt++/MltPlaylist.cpp
@@ -71,7 +71,7 @@ void ClipInfo::update( mlt_playlist_clip_info *info )
 	producer = new Producer( info->producer );
 	cut = new Producer( info->cut );
 	start = info->start;
-	resource = strdup( info->resource );
+	resource = info->resource ? strdup( info->resource ) : 0;
 	frame_in = info->frame_in;
 	frame_out = info->frame_out;
 	frame_count = info->frame_count;
